google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -508,7 +508,7 @@ public void write(Iterable<LogEntry> logEntries, WriteOption... options) {
   private static ListLogEntriesRequest listLogEntriesRequest(LoggingOptions serviceOptions,
       Map<Option.OptionType, ?> options) {
     ListLogEntriesRequest.Builder builder = ListLogEntriesRequest.newBuilder();
-    builder.addProjectIds(serviceOptions.getProjectId());
+    builder.addResourceNames("projects/" + serviceOptions.getProjectId());
```

Comment:
@michaelbausor could you give guidance on using the right resource name type here?